packages/core/src/types/arazzo.ts

```diff
@@ -0,0 +1,345 @@
+const arazzoSourceDescriptionSchema = {
+  type: 'object',
+  properties: {
+    name: { type: 'string' },
+    type: { type: 'string', const: 'arazzo' },
+    url: { type: 'string' },
+  },
+  additionalProperties: false,
+  required: ['name', 'type', 'url'],
+} as const;
+const sourceDescriptionSchema = {
+  type: 'object',
+  oneOf: [
+    openAPISourceDescriptionSchema,
+    noneSourceDescriptionSchema,
+    arazzoSourceDescriptionSchema,
+  ],
+} as const;
+const sourceDescriptionsSchema = {
+  type: 'array',
+  items: sourceDescriptionSchema,
+} as const;
+const extendedOperation = {
+  type: 'object',
+  properties: {
+    path: { type: 'string' },
+    method: operationMethod,
+    sourceDescriptionName: { type: 'string' },
+    serverUrl: { type: 'string' },
+  },
+  additionalProperties: false,
+  required: ['path', 'method'],
+} as const;
+const parameter = {
+  type: 'object',
+  properties: {
+    in: { type: 'string', enum: ['header', 'query', 'path', 'cookie'] },
+    name: { type: 'string' },
+    value: {
+      oneOf: [{ type: 'string' }, { type: 'number' }, { type: 'boolean' }],
+    },
+    $ref: { type: 'string' },
+    style: { type: 'string' },
+    target: { type: 'string' },
+    required: { type: 'boolean' },
+    schema: {
+      type: 'object',
+      additionalProperties: true,
+    },
+    example: {
+      type: 'object',
+      additionalProperties: true,
+    },
+    examples: {
+      oneOf: [{ type: 'object' }, { type: 'object', properties: { $ref: { type: 'string' } } }],
+    },
+  },
+  additionalProperties: false,
+  required: ['in', 'name', 'value'],
+} as const;
+const parameters = {
+  type: 'array',
+  items: parameter,
+} as const;
+const infoObject = {
+  type: 'object',
+  properties: {
+    title: { type: 'string' },
+    description: { type: 'string' },
+    summary: { type: 'string' },
+    version: { type: 'string' },
+  },
+  additionalProperties: false,
+  required: ['title', 'version'],
+} as const;
+const replacement = {
+  type: 'object',
+  properties: {
+    target: { type: 'string' },
+    value: {
+      oneOf: [
+        { type: 'string' },
+        { type: 'object' },
+        { type: 'array' },
+        { type: 'number' },
+        { type: 'boolean' },
+      ],
+    },
+  },
+} as const;
+const requestBody = {
+  type: 'object',
+  properties: {
+    contentType: { type: 'string' },
+    payload: {
+      oneOf: [
+        { type: 'string' },
+        { type: 'object', additionalProperties: true },
+        { type: 'array' },
+        { type: 'number' },
+        { type: 'boolean' },
+      ],
+    },
+    encoding: { type: 'string' },
+    replacements: {
+      type: 'array',
+      items: replacement,
+    },
+  },
+  additionalProperties: false,
+  required: ['payload'],
+} as const;
+const criteriaObject = {
+  type: 'object',
+  properties: {
+    condition: { type: 'string' },
+    context: { type: 'string' },
+    type: { type: 'string', enum: ['regex', 'JSONPath'] },
```

Comment:
I'm seeing examples that have this set to `jsonpath`, do you think we should use a case-insensitive match here to accept those?

Reply:
Its from Arazzo description, so I guess we should keep it as it is

```
    - context: $response.body
      condition: $[?count(@.pets) > 0]
      type: JSONPath
 ```

Reply:
I will fix their example instead ;)

Reply:
I just discovered it was change at some point in time - so you are right. Updating schemas....